### PR TITLE
feat: add header auth

### LINKED ISSUE #34: Is it possible to authenticate with a service account?
with the bearer token to be more specific, to set as the Authorization header instead of the basic auth

### PR DESCRIPTION
**Pull request by freak12techno**
Probably fixes #34.

### CODE REVIEW

**Comment by RodrigoPina407** [comment]
I think having the user and password defaults as "admin" is causing issues and it does not use the Auth token

File: pkg/config/config.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ type GrafanaConfig struct {
 	URL           string            `default:"http://localhost:3000"                                 yaml:"url"`
 	User          string            `default:"admin"                                                 yaml:"user"`
```
Comment (RodrigoPina407) [inline pkg/config/config.go:28]: remove default

**Comment by freak12techno** [comment]
@RodrigoPina407 actually yes, haven't thought of that, gonna fix it in a few minutes

**Comment by freak12techno** [comment]
@RodrigoPina407 can you check with the latest changes?

**Review by RodrigoPina407 — APPROVED** [review]
working well on my side

**Pull request merged**